### PR TITLE
upgrade modify

### PR DESCRIPTION
**Pull request by Balaspba**
complited

### CODE REVIEW

File: controllers/signup_controller.go [hunk 1]
```diff
@@ -1,5 +1,32 @@
 package controllers
 
-func () SignUp() {
```
Comment (RAJKUMAR-MADHIYAN) [inline controllers/signup_controller.go:3]: resolve conflict

File: database/postgresql.go [hunk 2]
```diff
@@ -12,10 +12,8 @@ import (
 var db *sql.DB
 
 // PostgreSQL connection
-func DBConnect(cfg *config.AppConfig) *sql.DB {
```
Comment (RAJKUMAR-MADHIYAN) [inline database/postgresql.go:15]: resolve conflict

**Review by RAJKUMAR-MADHIYAN — COMMENTED** [review]
resolve conflict

**Comment by Balaspba** [comment]
hello

**Pull request merged**